packages/pluggableWidgets/accordion-native/src/components/AccordionGroup.tsx

```diff
@@ -0,0 +1,82 @@
+import { createElement, ReactElement, useEffect } from "react";
+import { View, Pressable, Text } from "react-native";
+import { isAvailable } from "@mendix/piw-utils-internal";
+import { DynamicValue, NativeIcon } from "mendix";
+
+import { GroupIcon } from "./GroupIcon";
+import { AnimatedCollapsibleView } from "./CollapsibleView";
+import { GroupsType, IconEnum } from "../../typings/AccordionProps";
+import { AccordionGroupStyle } from "../ui/Styles";
+
+export interface AccordionGroupProps {
+    index: number;
+    collapsible: boolean;
+    icon: IconEnum;
+    iconCollapsed: DynamicValue<NativeIcon> | undefined;
+    iconExpanded: DynamicValue<NativeIcon> | undefined;
+    group: GroupsType;
+    isExpanded: boolean;
+    expandGroup: (index: number) => void;
+    onPressGroupHeader: (group: GroupsType, index: number) => void;
+    visible: DynamicValue<boolean>;
+    style: AccordionGroupStyle;
+}
+
+export function AccordionGroup({
+    index,
+    collapsible,
+    icon,
+    iconCollapsed,
+    iconExpanded,
+    group,
+    isExpanded,
+    expandGroup,
+    onPressGroupHeader,
+    visible,
+    style
+}: AccordionGroupProps): ReactElement | null {
+    useEffect(() => {
+        if (group.groupAttribute && isAvailable(group.groupAttribute) && group.groupAttribute?.value === false) {
+            expandGroup(index);
+        }
+    }, [group.groupAttribute]);
+
+    useEffect(() => {
+        if (
+            group.groupCollapsedDynamic &&
+            isAvailable(group.groupCollapsedDynamic) &&
+            group.groupCollapsed === "groupStartDynamic" &&
+            group.groupCollapsedDynamic?.value === false
+        ) {
+            expandGroup(index);
```

Comment:
Similar to above, howcome this effect doesn't also collapse based on the expression?